source/jormungandr/tests/isochrone_tests.py

```diff
@@ -90,12 +90,16 @@ def test_to_isochrone_coord(self):
     def test_from_isochrone_sa(self):
         query = "v1/coverage/basic_routing_test/journeys?from={}&datetime={}"
         query = query.format("stopA", "20120614T080000")
-        response = self.query(query)
+        normal_response, error_code = self.query_no_assert(query)
+        assert normal_response['error']['message'] == 'The entry point: stopA is not valid'
```

Comment:
The response of the type "no_solution" does not raise error while using "response = self.query(query)" in the test. 

After this correction we have "unknown_object" in the response which is not supported by "response = self.query(query)" in the test.